en_US/changes/changes-4.3.md

```diff
@@ -15,6 +15,23 @@ ref: undefined
 
 # Changes
 
+## Version 4.3.18
+
+*Release Date: 2022-08-11*
+
+### Important Changes
+
+- From the next release, we will stop supporting macOS 10 and provide an installation package for macOS 11
```

Comment:
https://github.blog/changelog/2022-07-20-github-actions-the-macos-10-15-actions-runner-image-is-being-deprecated-and-will-be-removed-by-8-30-22/

Maybe we should declare the reason why we stop supporting macOS 10.